Model/CommentList.java

```diff
@@ -78,11 +88,16 @@ public CommentModel getItem(String id) {
         }
         return null;
     }
+
     @Override
     public void setList() {
         // TODO Auto-generated method stub
         
     }
+    
+    /**
+     * A method that saves the commentList into the database comments.json
+     */
     @Override
     public void save() {
```

Comment:
please avoid calling for Mapper, just use the JsonParser that you have already intialized.

Reply:
resolved